inginious/frontend/plugins/manual_scoring/static/json/rubric_es.json

```diff
@@ -0,0 +1,87 @@
+{

```

Comment:
I don't see that this is used or is it? Otherwise try to load this json when the Spanish is the language of the user.